lib/kafka-shared/Makefile

```diff
@@ -21,7 +22,7 @@ init: certs
 	terraform init \
 		-backend-config region=eu-west-1 \
 		-backend-config bucket=uw-${ENV}-pubsub-tf-applier-state \
-		-backend-config key="${CONTEXT}"/kafka-shared \
+		-backend-config key="${CONTEXT}"/kafka-shared-${SUBPROJECT} \
```

Comment:
Might be worth a comment in README/k8s repo that the s3 bucket key is expected to take this format

Reply:
Not really sure where I would put this TBH